src/Akeneo/Category/front/src/feature/components/EditAttributesForm.tsx

```diff
@@ -140,15 +140,17 @@ export const EditAttributesForm = ({attributeValues, template, onAttributeValueC
     );
   });
 
+  const channelList = Object.values(channels);
```

Comment:
Here the channelList will have a new reference in memory at each render of the component. As you are using channelList as a props, it means ChannelSelector will be rerendered each time even if the values have not changed.
To avoid this case you can use the hook [useMemo](https://reactjs.org/docs/hooks-reference.html#usememo)

```suggestion
  const channelList = useMemo(() => Object.values(channels), [channelList]);
```